core/node/node_storage_init/src/external_node/revert.rs

```diff
@@ -22,8 +22,14 @@ impl RevertStorage for ExternalNodeReverter {
         to_batch: L1BatchNumber,
         _stop_receiver: watch::Receiver<bool>,
     ) -> anyhow::Result<()> {
+        let Some(block_reverter) = self.reverter.as_ref() else {
+            anyhow::bail!(
+                "Revert to block {to_batch} was requested, but the reverter was not provided."
+            );
+        };
```

Comment:
Nit: This can be more concisely expressed by using `self.reverter.as_ref().context("...")?`.